x-pack/plugins/licensing/server/index.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { PluginInitializerContext } from 'src/core/server';
+import { schema } from './schema';
+import { LicensingPlugin } from './licensing_plugin';
```

Comment:
nit: [by convention](https://github.com/elastic/kibana/blob/master/src/core/CONVENTIONS.md), this file should be called "plugin"